src/registry.json

```diff
@@ -569,6 +569,33 @@
           "checksum": "hWHPH9JldbbLDgIYWj0i4rUs2BSwNbYANYivMI30fqA="
         }
       }
+    },
+    "npm:@partisiablockchain/snap": {
+      "id": "npm:@partisiablockchain/snap",
+      "metadata": {
+        "name": "Partisia Blockchain",
+        "author": {
+          "name": "Partisia Blockchain",
+          "website": "https://partisiablockchain.com/"
+        },
+        "website": "https://partisiablockchain.gitlab.io/documentation/smart-contracts/integration/metamask-snap-integration.html",
+        "summary": "Snap to interact with Partisia Blockchain",
+        "description": "The Partisia Blockchain MetaMask snap allows dapp developers to use MetaMask for signing transactions towards Partisia Blockchain.",
+        "audits": [
+          {
+            "auditor": "Veridise",
+            "report": "https://f8t2x8b2.rocketcdn.me/wp-content/uploads/2023/09/VAR-Partisia-Blockchain-Snap.pdf"
```

Comment:
Should we re-host the audit file like we've done in other PRs?

Reply:
This is hosted by the auditor and I will opt to do that as much as possible, either pointing to auditor's site or developer's site, but we have backups in case we need to point to our own file in the future.